src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -180,4 +180,5 @@ private-f net.minecraft.world.server.ChunkHolder field_219320_o # block update l
 public net.minecraft.world.server.ServerChunkProvider field_186029_c # chunkGenerator
 public net.minecraft.world.server.ServerChunkProvider field_73251_h # worldObj
 private-f net.minecraft.world.storage.loot.LootPool field_186455_c # rolls
-private-f net.minecraft.world.storage.loot.LootPool field_186456_d # bonusRolls
\ No newline at end of file
+private-f net.minecraft.world.storage.loot.LootPool field_186456_d # bonusRolls
+public-f net.minecraft.world.raid.Raid$WaveMember field_221284_f # VALUES
```

Comment:
Why is the VALUES field make publicly setable?
Are we intentionally letting random people screw with other waves entries?
Would it be better to have a add/removeMemeber function?